generators/chipyard/src/main/scala/config/fragments/PeripheralFragments.scala

```diff
@@ -29,7 +29,7 @@ import chipyard.{ExtTLMem}
   * @param hang the power-on reset vector, i.e. the program counter will be set to this value on reset
   * @param contentFileName the path to the BootROM image
   */
-class WithBootROM(address: BigInt = 0x10000, size: Int = 0x10000, hang: BigInt = 0x10040) extends Config((site, here, up) => {
+class WithBootROM(address: BigInt = 0x10000, size: Int = 0x10000, hang: BigInt = 0x10000) extends Config((site, here, up) => {
```

Comment:
If this is changed, then we need to change the testchipip bootrom as well. Does this PR have a corresponding testchipip PR that changes that? 

Also, what is the motivation for this change?